hack/dump-debug-info.sh

```diff
@@ -8,7 +8,7 @@ echo "dumping debug information"
 
 echo "Running ocs-must-gather to ${OCS_MUST_GATHER_DIR}"
 mkdir -p "$OCS_MUST_GATHER_DIR"
-${OCS_OC_PATH} adm must-gather --image "$MUST_GATHER_FULL_IMAGE_NAME" --dest-dir "$OCS_MUST_GATHER_DIR"
+${OCS_OC_PATH} adm must-gather --image "$OCS_MUST_GATHER_IMAGE" --dest-dir "$OCS_MUST_GATHER_DIR"
```

Comment:
The image here should be "$LATEST_MUST_GATHER_IMAGE", As this script is just sourcing from common.sh, And there it's defined as such.